Edits (#2)
krishnangovindraj/typedb-docs: new_core_concepts/modules/ROOT/pages/typeql/constraining-data.adoc

= Constraining Data

TypeDB allows additional constraints to be defined on the types in the schema.

> These constraints can restrict the cardinality of an interface, or the range of values of an attribute-type.

Reviewer: I think I prefer domain since range is overloaded by @range

> These constraints are written using *annotations* in TypeQL. Annotations are written with `@` and apply to schema declarations. Note that there are other types of annotations besides data constraint annotations as well.

Reviewer: ```suggestion
These constraints are written as *annotations* on the schema declarations they apply to. Annotations are prefixed with the `@` symbol
```
